### DISCUSSION TITLE
Errore nell'impostazione database MySQL phpMyAdmin

### DISCUSSION THREAD

**Discussion by hyperm03** [iisgiua/giuaschool#457] [Richieste di aiuto] [answered]
Buongiorno.

Sono uno studente, e per fini di tipo sperimentale stavo provando giuaschool su server web XAMPP con MySQL PhpMyAdmin, in esecuzione su client Wndows 11. Nonostante imposto correttamente i settaggi del database durante l'installazione, mi da un errore di impossibilità a connettersi. Aggiungo anche che mi da un Fatal error in basso, sempre nella pagina di inserimento dettagli del database, che dice quanto segue:

Fatal error: Uncaught TypeError: Invalid return value: callable object expected, "int" returned from "C:/xampp/htdocs/guia/public/install/index.php". in C:\xampp\htdocs\guia\vendor\autoload_runtime.php:18 Stack trace: #0 C:\xampp\htdocs\guia\public\install\index.php(4): require_once() #1 {main} thrown in C:\xampp\htdocs\guia\vendor\autoload_runtime.php on line 18

Attendo vostro riscontro

**Comment by trinko** [comment, marked as answer]
Assicurati di aver eseguito le istruzioni indicate nella documentazione, in particolare controlla i requisiti di sistema e dove scaricare la versione del codice:
[installazione](https://iisgiua.github.io/giuaschool-docs/install.html)

Inoltre, visto che si tratta di una installazione di prova, ti consiglio di utilizzare i docker (una sorta di macchina virtuale): vedi nella documentazione i riferimenti.
